content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,53 @@
+| **SLO details side panel (up to 90 days of historical data)** | Can set custom time windows to view SLO info                                                                                              | Cannot set custom time windows to view SLO info (can view 7, 30, or 90 day history)                                                                                                                     | Can set custom time windows to view SLO info                                         |
+| **SLO alerting ([Error Budget][1] or [Burn Rate][2] Alerts)**         | Available                                                                                                      | Available for SLOs based on Metric Monitor types only (not available for Synthetic Monitors or Service Checks)                                                      | Not available                                                    |
+| [**SLO Status Corrections**][3]                                       | Correction periods are ignored from SLO status calculation                                                              | Correction periods are ignored from SLO status calculation                                                                                        | Correction periods are counted as uptime in SLO status calculation |
+| **[SLO Widgets][4] (up to 90 days of historical data)**                                                  | Available                                                                                                           | Available                                                                                                                           | Available                                                        |
+| [**SLO Data Source**][5]                                              | Available (with up to 15 months of historical data)                                                                | Not available                                                                                                                                     | Not available                                                |
```

Comment:
For time slices can we say "Coming soon".

Reply:
We try to stay away from future promises in docs. We can update the docs as soon as features are available!